reporter/src/com/intellij/rt/coverage/report/Reporter.java

```diff
@@ -50,12 +50,13 @@ public void createXMLReport(File xmlFile) throws IOException {
     }
   }
 
-  public void createHTMLReport(File htmlDir, String title) throws IOException {
+  public void createHTMLReport(File htmlDir, String title, String charset) throws IOException {
     htmlDir.mkdirs();
     final HTMLReportBuilder builder = ReportBuilderFactory.createHTMLReportBuilderForKover();
     builder.setReportDir(htmlDir);
     if (builder instanceof HTMLReportBuilderImpl) {
       ((HTMLReportBuilderImpl) builder).setReportTitle(title);
+      ((HTMLReportBuilderImpl) builder).setCharset(charset);
```

Comment:
`HTMLReportBuilderImpl` internally has a default value that should fit most of the cases `Charset.defaultCharset().displayName()`. So, I think that it is better to change it only when `charset != null`. Something like this:
```
if (charset != null) {
  ((HTMLReportBuilderImpl) builder).setCharset(charset);
}
```

Reply:
you're right. i'll change it!